README.md

```diff
@@ -53,24 +96,30 @@ const response = await fetch(poetNodeUrl + '/works/', {
 })
 ```
 
-Note, if you are creating an identity claim, your Identity Provider (IDP) <!-- TODO: link to glossary --> will be the issuer of the claim. If you are self-serving your own IDP, you will have to create an IdentityClaim for the IDP from which you can issue all further identities.
+### Example 2: createClaim for Work Claim, with overriding context
+
+Note, if you are creating an identity claim, your IDP will be the issuer of the claim. [Frost](https://frost.po.et/) is one such IDP.
+If you are self-serving your own identity calim, your identity provider (IDP) will have to create an IdentityClaim for 
+itself from which you can issue all further identities. Currently the Po.et network uses the [Ed25519Signature2018](https://w3c-dvcg.github.io/lds-ed25519-2018/), 
+which requires a Base58 form of the Ed25519 Public Key.
 
 ### Example 2: createClaim for Identity Claims <!-- TODO: link to glossary -->
 
-```ts
-import { Claim, ClaimType, createClaim } from '@po.et/poet-js'
+```typescript
+import { Claim, ClaimType, createClaim, KeyHelper } from '@po.et/poet-js'
+
+// Store the privateKey for signing future claims
+const { publicKey, privateKey } = KeyHelper.generateED25519Base58Keys('password')
 
 const identityAttributes = {
-  publicKey: ''
+  publicKey,
 }
 
-const Issuer = 'po.et://entities/<idp identity claim id>'
-
 const claim = createClaim(
-  Issuer,
   ClaimType.Identity,
-  identityAttributes
-) 
+  identityAttributes,
+  claimSigner,
```

Comment:
claimSigner is not defined in this example

Reply:
Oh, nvm, I see you were using the above claim signer for all the examples. That is sort of confusing because you are generating new keys in this example

Reply:
The new key is for the new identity whereas the `claimSigner` is the issuer of that claim.

However I added claimSigner into each of the examples for clarity.

Reply:
Cool, yeah that helps 👍